src/App.css

```diff
@@ -1,38 +1,329 @@
+  text-align: left;
+  color: white;
+}
+
+nav p {
+  width: 25%;
+  padding: 1rem 0;
+}
+
+ul, li {
+  padding: 0;
+  margin: 0;
+}
+
+/* header */
+.header {
+  height: 30vh;
   display: flex;
   flex-direction: column;
+  margin-top: 3rem;
   align-items: center;
+  row-gap: 1.5rem;
+  justify-content: flex-end;
+}
+
+/* image width */
+img {
+  width: 100%;
+  border-radius: 0.5rem;
+  box-shadow: 2px 2px 2px grey;
+}
+
+span {
+  font-weight: 700;
+}
+
+/* input form */
+form {
+  width: 80%;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+input, button {
+  border-radius: 0;  
+  padding: 0.5rem 0;
+}
+
+input {
+  width: 70%;
+  border: 1px solid grey;
+  padding-left: 0.75rem;
+  border-radius: 1.5rem 0 0 1.5rem;
+}
+
+button {
+  width: 30%;
+  font-family: 'DM Sans', sans-serif;
+  background-color: #723C4F;
   color: white;
+  border: 1px solid #723C4F; 
+  border-radius: 0 1.5rem 1.5rem 0;
+}
+
+button:hover, button:focus {
+  background-color: #8c435d;
 }
 
-.App-link {
-  color: #61dafb;
+/* main art display */
+main {
+  padding: 1.25rem 0;
+}
+
+main > p {
+  color: #656565;
+  margin: 2rem 0;
+}
+
+.images h2 {
+  flex-basis: 100%;
+  text-align: left;
+  padding: 2rem 0;
+}
+
+/* art */
+.artDisplay {
+  background-color: #f2ebe3;
+  text-align: left;  
+  padding: 2rem;
+  display: flex;
+  column-gap: 3rem;
+  border-radius: 0.5rem;
+}
+
+.imgContainer,
+.description {
+  width: 50%;
+}
+
+.description h2 {
+  font-size: 1.8rem;
+  margin: 2rem 0;
+}
+
+h2 ~ p {
+  line-height: 1.3;
+  margin-top: 0.25rem;
+}
+
+/* tags in results */
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 0.5rem;
+  row-gap: 0.5rem;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+.tags > p:first-child {
+  font-weight: 700;
+  margin: 0.5rem 0;
+  flex-basis: 100%;
+}
+
+.tags li {
+  text-transform: uppercase;
+  font-size: 0.8rem;
+  letter-spacing: 1px;
+  list-style: none;
+  border: 1px solid rgb(144, 93, 5);
+  color: rgb(144, 93, 5);
+  padding: 0.5rem 1rem;
+  border-radius: 15px;
+}
+
+/* saved section */
+div.saved {
+  background-color: none;
+  width: 100%;
+}
+
+.saved ul {
+  padding-left: 0;
+  display: flex;
+  column-gap: 1rem;
+  row-gap: 1rem;
+  flex-wrap: wrap;
+}
+
+.saved li {
+  width: calc((100% / 6) - 1rem);
+  display: inline-block;
+  list-style: none;
+}
+
+.saved img {
+  height: 300px;
+  object-fit: cover;
+  border-radius: 20px;
+}
+
+.remove-btn,
+.save-btn {
+  border-radius: 2rem;
+  width: 100%;
+}
+
+.save-btn {
+  position: relative;
+  right: 0;
+  top: 0;
+  width: 30%;
+}
+
+.remove-btn {
+  margin-top: 1rem;
+}
+
+/* footer general styles */
+.footer {
+  margin-top: 3rem;
+  border-top: 1px solid black;
+  padding: 1rem;
+}
+
+footer p {
+  margin-top: 0.5rem; 
+}
+
+footer a {
+  text-decoration: none;
+  color: inherit;
+}
+
+@media (max-width: 940px) {
```

Comment:
media queries look great as well - it's a very smooth transition to small viewport, still very readable.